packages/js/src/settings/routes/site-representation.js

```diff
@@ -44,6 +45,11 @@ const SiteRepresentation = () => {
 	const mastodonPremiumLink = useSelectSettings( "selectLink", [], "https://yoa.st/get-mastodon-integration" );
 	const mastodonUrlLink = useSelectSettings( "selectLink", [], "https://yoa.st/site-representation-mastodon" );
 
+	const handleAddProfile = useCallback( async( arrayHelpers ) => {
+		await arrayHelpers.push( null, "" );
+		document.getElementById( `input-wpseo_social-other_social_urls-${ otherSocialUrls.length }` )?.focus();
```

Comment:
Please do not combine React rendering with vanilla element getting like this and use a `ref` instead. While it should be fine with just trying with that null check, we can just rely on React to let us know the element, which will populate when it rendered.

Possibly this is[ the `innerRef` prop of the Formik Field](https://formik.org/docs/api/field#innerref)? I'm not sure 😁